sdk/avalon_sdk/worker/worker_request_validator.py

```diff
@@ -53,24 +40,36 @@ def worker_lookup_validation(self, params):
         if len(params) == 0:
             return False, "Empty params in the request"
 
-        key_list = []
-        for key in params.keys():
-            if key not in self.__lookup_param_key_map.keys():
-                return False, "Invalid parameter {}".format(key)
+        schema = {
+            "type": "object",
+            "uniqueItems": True,
+            "additionalProperties": False,
+            "properties": {
+                "workerType": {
+                    "type": "integer",
+                    "enum": [1, 2, 3],
+                    "error_msg":
+                        "Invalid data format for workerType"},
+                "organizationId": {
+                    "type": "string",
+                    "pattern": "[0-9xXa-fA-F]",
+                    "error_msg":
+                        "Invalid data format for organizationId"},
+                "applicationTypeId": {
+                    "type": "string",
+                    "pattern": "[0-9xXa-fA-F]",
+                    "error_msg":
+                        "Invalid data format for applicationTypeId"}
+            }
+        }
+
+        try:
+            validate(params, schema)
+        except ValidationError as e:
```

Comment:
Can add another except for SchemaError as well?

Reply:
Done